code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/ProjectMemberResourceManager.java

```diff
@@ -0,0 +1,123 @@
+package io.cattle.platform.iaas.api.auth.github;
+
+import io.cattle.platform.api.auth.ExternalId;
+import io.cattle.platform.api.auth.Policy;
+import io.cattle.platform.api.resource.AbstractObjectResourceManager;
+import io.cattle.platform.core.dao.GenericResourceDao;
+import io.cattle.platform.core.model.Account;
+import io.cattle.platform.core.model.ProjectMember;
+import io.cattle.platform.iaas.api.auth.ExternalIdHandler;
+import io.cattle.platform.iaas.api.auth.dao.AuthDao;
+import io.cattle.platform.iaas.api.auth.github.resource.Member;
+import io.cattle.platform.json.JsonMapper;
+import io.cattle.platform.object.ObjectManager;
+import io.cattle.platform.object.process.ObjectProcessManager;
+import io.github.ibuildthecloud.gdapi.context.ApiContext;
+import io.github.ibuildthecloud.gdapi.exception.ClientVisibleException;
+import io.github.ibuildthecloud.gdapi.factory.SchemaFactory;
+import io.github.ibuildthecloud.gdapi.model.ListOptions;
+import io.github.ibuildthecloud.gdapi.request.ApiRequest;
+import io.github.ibuildthecloud.gdapi.util.ResponseCodes;
+
+import java.util.*;
+
+import javax.inject.Inject;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class ProjectMemberResourceManager extends AbstractObjectResourceManager {
+
+    @Inject
+    JsonMapper jsonMapper;
+    @Inject
+    AuthDao authDao;
+    @Inject
+    GenericResourceDao resourceDao;
+    @Inject
+    ObjectManager objectManager;
+    @Inject
+    ObjectProcessManager objectProcessManager;
+
+    private List<ExternalIdHandler> externalIdHandlers;
+
+    @Override
+    protected Object removeFromStore(String type, String id, Object obj, ApiRequest request) {
+        return null;
```

Comment:
If this is an unimplemented function, I'm tempted to say throw an exception here to fail fast for anyone who happens to call it in the future.  I haven't investigated more to trace back who calls this or what the repercussions might be.